contracts/sol6/IKyberFeeHandler.sol

```diff
@@ -10,10 +10,12 @@ interface IKyberFeeHandler {
     event KncBurned(uint256 kncTWei, IERC20 indexed token, uint256 amountTwei);
 
     function handleFees(
+        IERC20 token,
```

Comment:
should we use `Twei` here as well?

Reply:
`platformFeeTwei` and `feeBRRTwei`? Mmmm...

Reply:
The interface APIs are designed for both eth + token as fee right?

Reply:
for Eth we need wei
for token we need twei.
can't have them both.

we could call one:
platformFee
the other
networkFee

and in comment say it is wei for eth, twei for token.

Reply:
Then for event we also need to change amount name?

Reply:
not sure.

Reply:
Ok, will change.